src/lib/utils/meta_table_manager.cpp

```diff
@@ -90,12 +90,18 @@ auto gather_segment_meta_data(const std::shared_ptr<Table>& meta_table, const Me
 namespace opossum {
 
 MetaTableManager::MetaTableManager() {
-  _methods["tables"] = &MetaTableManager::generate_tables_table;
-  _methods["columns"] = &MetaTableManager::generate_columns_table;
-  _methods["chunks"] = &MetaTableManager::generate_chunks_table;
-  _methods["chunk_sort_orders"] = &MetaTableManager::generate_chunk_sort_orders_table;
-  _methods["segments"] = &MetaTableManager::generate_segments_table;
-  _methods["segments_accurate"] = &MetaTableManager::generate_accurate_segments_table;
+  _methods["tables"] = {&MetaTableManager::generate_tables_table,
+                              &MetaTableManager::generate_tables_table_stub};
```

Comment:
Any reason not to use a boolean parameter that tells the method to fill the table with data (or not)?